### PR TITLE
feat: 0.64 release

### PR DESCRIPTION
**Pull request by grabbou**
Article for the 0.64 release.

### CODE REVIEW

**Comment by grabbou** [comment]
Link to the article itself is here: https://deploy-preview-2538--react-native.netlify.app/blog/2021/03/11/version-0.64

File: website/blog/2021-03-11-version-0.64.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+title: Announcing React Native 0.64 with Hermes on iOS
+author: Mike Grabowski
+authorTitle: CTO and Co-Founder at Callstack
+authorURL: 'https://twitter.com/grabbou'
+authorImageURL: 'https://avatars0.githubusercontent.com/u/2464966?s=460&v=4'
+authorTwitter: grabbou
+tags: [announcement, release]
+---
+
+Today we’re releasing React Native 0.64 that ships with LogBox turned on by default.
```
Comment (TheSavior) [inline website/blog/2021-03-11-version-0.64.md:11]: Copy paste error? This release doesn't have LogBox changes

File: website/blog/2021-03-11-version-0.64.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+```jsx
+function MyComponent(props) {
+  const result = require('my-module').MyFunction;
+
+  return (<Text>{result}</Text>);
+}
+```
+
+More information about Inline Requires is available in the [Performance documentation](https://reactnative.dev/docs/ram-bundles-inline-requires#inline-requires).
```
Comment (Simek) [inline website/blog/2021-03-11-version-0.64.md:63]: All the internal links should not include domain.

**Review by Simek — APPROVED** [review]
@grabbou Great work on this release! 👏

**Pull request merged**